field/time.go

```diff
@@ -116,41 +116,38 @@ func (nt *NullTime) Scan(value interface{}) error {
 	case time.Time:
 		if v.IsZero() {
 			nt.Valid = false
-			nt.validNull = true
+			nt.invalidNull = true
```

Comment:
Good question, this checks to see if it is a legitimate Zero time, 12AM which should not be considered...
Ok, now I see I need to change this. Probably remove the Zero Check altogether.